tests/ui/invalid_pyfunctions.rs

```diff
@@ -15,4 +15,7 @@ fn wildcard_argument(_: i32) {}
 #[pyfunction]
 fn destructured_argument((a, b): (i32, i32)) {}
 
+#[pyfunction]
+fn function_with_required_after_option(_opt: Option<i32>, _x: i32) {}
```

Comment:
I think this would be good to mention in the changelog as well as it did cause quite a bit of confusion when we initially changed this IIRC. So I think at least some users will just have silenced the deprecation warning.

Reply:
Mentioned in the migration guide.